driver/src/deposit_driver.rs

```diff
@@ -41,12 +41,7 @@ pub fn run_deposit_listener<D, C>(db: &D, contract: &C) -> Result<(bool), Driver
 
             let deposits = db.get_deposits_of_slot(slot.low_u32())?;
             let deposit_hash = deposits.rolling_hash(0);
-            if deposit_hash != contract_deposit_hash {
-                return Err(DriverError::new(
-                    &format!("Pending deposit hash from contract ({}), didn't match the one found in db ({})", 
-                    contract_deposit_hash, deposit_hash), ErrorKind::StateError
-                ));
-            }
+            check_consistency_of_hashes(deposit_hash, contract_deposit_hash, &(String::from("deposit")) )?;
```

Comment:
Why do you need to create a string and then reference it? Shouldn't 

```
check_consistency_of_hashes(deposit_hash, contract_deposit_hash, "deposit")?;
```

work?

Reply:
yes it does!